recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ build:
   entry_points:
     - nuitka = nuitka.__main__:main
     - nuitka-run = nuitka.__main__:main
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py==37]
```

Comment:
I am guessing that the part where I have this:

```
requirements:
  host:
    - python
    - libpython-static
    - pip
  run:
    - python
    - libpython-static
```

can be fixed in the same way, e.g.

```
requirements:
  host:
    - python
    - libpython-static # [py!=37]
    - pip
  run:
    - python
    - libpython-static # [py!=37]
```

What I would like to have is an optional requirement, it seems for some Python versions, the static libpython is not present or broken. Can I just not error out if something I write there does not exist?

Reply:
if that is done, `skip: true  # [py==37]` can be dropped I assume.